notebooks/229-distilbert-sequence-classification/data/sample.txt

```diff
@@ -0,0 +1,2 @@
+The food was horrible.
```

Comment:
@arunimaaaa according the latest changes, data for all notebooks shuld be placed in notebooks/data directory, please move your sample.txt to `notebooks/data/text` and give more descriptive name

Reply:
done